spanner/google/cloud/spanner/__init__.py

```diff
@@ -15,6 +15,10 @@
 """Cloud Spanner API package."""
 
 
+import pkg_resources
+__version__ = pkg_resources.get_distribution('google-cloud-spanner').version
```

Comment:
@lukesneeringer I considered doing this for all sub-packages. Seems you are :+1: on this approach?

Also: defining `__version__` should come **after** all the imports?